src/py21cmfast/src/UsefulFunctions.c

```diff
@@ -27,7 +27,8 @@ void seed_rng_threads(gsl_rng * rng_arr[], int seed){
     unsigned int seeds[user_params_ufunc->N_THREADS];
 
     // For multithreading, seeds for the RNGs are generated from an initial RNG (based on the input random_seed) and then shuffled (Author: Fred Davies)
-    int num_int = INT_MAX/16; //JD: this was taking a few seconds per snapshot so i reduced the number TODO: init the RNG once
+    // int num_int = INT_MAX/16;
+    int num_int = INT_MAX/256; //JD: this was taking a few seconds per snapshot so i reduced the number TODO: init the RNG once
```

Comment:
is the comment here still applicable?

Reply:
Yes, I had un-done this change in the initial v4-prep commit, but it takes a few seconds per snapshot. In a future commit it would be useful to only initialise the RNG once, along with some interpolation tables like sigma.

Reply:
For now I just reduced the maximum seed number